apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedNextSteps.md

```diff
@@ -14,7 +14,7 @@ The [Fluent UI React wiki](https://github.com/microsoft/fluentui/wiki) has infor
 
 Fluent UI React components are highly customizable using CSS-in-JS. We also support customization of colors and more throughout the entire app using themes.
 
-For an overview of styling and theming approaches in Fabric, see [this page](https://github.com/Microsoft/frontend-bootcamp/tree/master/step2-03/demo). Then check out the links below for more details:
+For an overview of styling and theming approaches in Fluent UI, see [this page](https://github.com/Microsoft/frontend-bootcamp/tree/master/step2-03/demo). Then check out the links below for more details:
```

Comment:
We should ideally also update the frontend-bootcamp code to use the new package (at least add a work item for that somewhere?)

Reply:
https://github.com/microsoft/frontend-bootcamp/issues/179